sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/ImageServiceImpl.java

```diff
@@ -38,26 +39,38 @@ public ImageDto getImage(Long id) {
         return image;
     }
 
-    public Long save(ImageDto image) {
+    public Long save(MultipartFile file) {
```

Comment:
As for my opinion. I assume that it is better to remove maven dependency from the backend antifactory:
```
<dependency>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-web</artifactId>
</dependency>
```
Backed part shouldn't know about web part. Instead of that we could create one more service `ImageClientService` which can handle `MultipartFile` object and call method from `ImageService` to save the image